Content.Client/IconSmoothing/IconSmoothComponent.cs

```diff
@@ -26,6 +26,12 @@ public sealed partial class IconSmoothComponent : Component
         [ViewVariables(VVAccess.ReadWrite), DataField("key")]
         public string? SmoothKey { get; private set; }
 
+        /// <summary>
+        ///     Additional keys to smooth with.
+        /// </summary>
+        [DataField]
+        public List<string> AdditionalKeys { get; private set; } = new();
```

Comment:
```suggestion
        public List<string> AdditionalKeys = new();
```
we don't need getters and setters for components anymore